include/database.py

```diff
@@ -59,3 +59,69 @@ def insert_returning(connectionString, sql, parameters):
     except psycopg2.DatabaseError as err:
         sys.stderr.write(f'A general database error occurred: {err}')
         raise DatabaseError
+
+
+def remove_record(connectionString, sql, parameters):
+    try:
+        # Open the connection, override any timeout provided with something shorter
+        # since we expect to be running interactively
+        connection = psycopg2.connect(connectionString, connect_timeout=1)
+        cursor = connection.cursor()
+
+        # Execute the query, note the rows
+        cursor.execute(sql, (parameters['id'],))
+
+        # Clean-up and return
+        connection.commit()
+        # check if the study exists
+        if cursor.rowcount:
+            print("Study deleted successfully")
+        else:
+            print("Study does not exist")
```

Comment:
While it's a good idea to return the number of rows impacted by the query, printing that information to `stdout` is generally a bad idea since the script calling this library doesn't know what happens. Better to just return the `cursor.rowcount` and let the caller decide what to do with that value.